pi-manage

```diff
@@ -559,6 +562,16 @@ except TypeError:
     profile = manager.command(profile)
 
 
+@authcache_manager.command
+def cleanup(minutes=480):
+    """
+    Remove entries from the authcache which last_auth entry is older than
```

Comment:
which -> where

Reply:
You see, I think I wanted to say "for which..."

Reply:
done in 4506a7bb